.travis.yml

```diff
@@ -254,24 +255,24 @@ before_script:
   - initctl list || true
 
   # CMake debug build
-  - cmake -H.
-    -BBuild-Debug
+  - cmake -S .
```

Comment:
Shouldn't Travis be using Meson instead of CMake?

Reply:
As Dennis said, Travis is using both